test/functional/Java16andUp/src/org/openj9/test/foreignMemoryAccess/TestCloseScope0.java

```diff
@@ -83,12 +83,25 @@
 	@Test(expectedExceptions=java.lang.IllegalStateException.class)
 	public static void closeScopeDuringAccess() throws Throwable {
 		/* Reflect setup */
-		Class c = Class.forName("jdk.internal.foreign.MemoryScope");
-		Method createShared = c.getDeclaredMethod("createShared", new Class[] {Object.class, Runnable.class, Cleaner.class});
-		createShared.setAccessible(true);
-		Method close = c.getDeclaredMethod("close");
+		Class MemoryOrResourceScope;
+		Method createShared;
+		Method close;
+		Object scope;
+
+		String version = System.getProperty("java.vm.specification.version");
```

Comment:
if you still need a version check, you can use the below API:
```suggestion
		int version = VersionCheck.major();
```